embulk-core/src/main/java/org/embulk/spi/util/Reporters.java

```diff
@@ -0,0 +1,78 @@
+package org.embulk.spi.util;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.embulk.spi.Reporter;
+import org.embulk.spi.ReporterCloseable;
+
+import java.util.Map;
+
+public final class Reporters
```

Comment:
Thank you. Make sense. They should not be in `org.embulk.spi`. I will move such methods in `Reporters` within BulkLoader.java.